report/src/presence_technologies/video_based_detection.tex

```diff
@@ -0,0 +1,34 @@
+\subsection{Video-based detection} \label{sec:video-based-detection}
+Cameras are a common technology used to detect the presence of humans in a room.
+There are many different approaches to how one might use a camera for presence detection.
+
+One of the primary methods for presence detection is the use of machine learning techniques.
+These techniques work by training a machine learning model, such as a convoluted neural network (CNN).
+The resulting model can then be used to analyze images and detect objects in them.
+For example, \citeauthor{FUERTES2022103473}\cite{FUERTES2022103473} used an omnidirectional camera to gather images of a room, and then trained a CNN to detect the presence of people in these images.
+The training is done through point-based annotation of the images as opposed to the more common approach of using bounding box annotation.
+Essentially, images are annotated with points denoting the location of the person in the image as opposed to drawing a bounding box around the person.
+This approach is advantageous because it is faster to annotate images in this way, but it is also far more accurate at representing the position of a human on images taken with an omnidirectional camera, which are often distorted or skewed.\cite{FUERTES2022103473}
+
+Motion detection is another commonly used to detect the presence of people in a room, and is defined by \citeauthor{ANANDAN1988347}\cite{ANANDAN1988347} as "the task of detecting movement of image elements across the image plane".
+In other words, motion detection is a broad term that can refer to a variety of different methods.
+For example, motion detection can be achieved by comparing two images of the same scene, and detecting the differences between them\cite{granath_detecting_nodate}.
+
+Alternatively, one may also utilize passive infrared (PIR) sensors to detect motion.
+PIR sensors are able to detect motion by measuring the infrared radiation emitted by objects in the room.\cite{Deiana2014}
+
+Many other methods exist, and the choice of method will ultimately depend on the specific application and system requirements. 
+Regardless of the method, a number of key considerations apply when using cameras for presence detection, including the sensitivity of the devices to lighting, obstructions, movement, temperature, and detection range and area.
+For example, if the device is obstructed by some object, it may be unable to gather useful data.
+For cameras specifically, consider how it may not be able to get a clear image of a room if the lighting is too dark or too bright.
+Most current systems use also standard perspective cameras with a narrow field of view that can only detect people in a limited area in front of the camera\cite{FUERTES2022103473}.
```

Comment:
```suggestion
Most current systems use standard perspective cameras with a narrow field of view that can only detect people in a limited area in front of the camera\cite{FUERTES2022103473}.
```

Reply:
It's mostly the ordering I disagree with.
```suggestion
Most current systems also use standard perspective cameras with a narrow field of view that can only detect people in a limited area in front of the camera\cite{FUERTES2022103473}.
```

Reply:
OK that was our mistake, agree. Didn't notice the order was wrong.